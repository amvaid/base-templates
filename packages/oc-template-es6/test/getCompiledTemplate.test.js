// const fs = require('fs');
// const getCompiledTemplate = require('../').getCompiledTemplate;

// test('Return compiled template when valid', () => {
//   const template = '';
//   const key = 'TBD';

//   expect(getCompiledTemplate(template, key).toString()).toMatchSnapshot();
// });

// test('Throw exception when not valid', () => {
//   const template = '';
//   const key = 'TBD';

//   expect(() =>
//     getCompiledTemplate(template, key)
//   ).toThrowErrorMatchingSnapshot();
// });

// test('Return undefined when key when valid', () => {
//   const template = '';
//   const key = 'TBD';

//   expect(getCompiledTemplate(template, key)).toBeUndefined();
// });

test('', () => {});
