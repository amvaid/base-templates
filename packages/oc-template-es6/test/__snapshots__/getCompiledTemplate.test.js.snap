// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Throw exception when not valid 1`] = `"Cannot read property 'components' of undefined"`;
